stac_fastapi/types/stac_fastapi/types/core.py

```diff
@@ -38,7 +38,7 @@ def create_item(
         collection_id: str,
         item: Union[Item, ItemCollection],
         **kwargs,
-    ) -> Optional[Union[Item, Response, None]]:
+    ) -> Optional[Union[stac.Item, Response, None]]:
```

Comment:
To Be Discussed! 

It seems weird to return a Pydantic Object, while non-transaction endpoints return `stac types` (typedDict) object

Reply:
I would agree - we don't want to return pydantic objects